qiita_db/data.py

```diff
@@ -1021,6 +1125,61 @@ class ProcessedData(BaseData):
     _preprocessed_processed_table = "preprocessed_processed_data"
 
     @classmethod
+    def get_by_status(cls, status):
```

Comment:
Since we have this for processed data, we should have it for preprocessed and raw data as well to keep things consistent. Anything that has a status really should have this classmethod.